frontend/src/app/register-form/register-form.component.ts

```diff
@@ -27,6 +27,7 @@ export class RegisterFormComponent {
       .registerUser({
         username: this.registerForm.controls.name.value,
         password: this.registerForm.controls.password.value,
+        email: this.registerForm.controls.email.value,
       })
```

Comment:
Could be just
```
.registerUser(this.registrerForm.value)
```

Reply:
I don't think so. Form has more fields like. repeat password

Reply:
Well, `registerUser` doesn't care about those fields and it wasn't even possible to reach them because Typescript would yell but anyway.

Reply:
Well. I changed it but UT failed. Just like I said, additional fields from `registerForm` cause problems
![image](https://user-images.githubusercontent.com/8255469/122990689-88506700-d3a4-11eb-9b13-a49b8d73ef2d.png)
![image](https://user-images.githubusercontent.com/8255469/122990628-78388780-d3a4-11eb-8468-1f7e7d227087.png)
![image](https://user-images.githubusercontent.com/8255469/122990530-56d79b80-d3a4-11eb-97d7-686b1fa97c99.png)
I don't think that we should expand `RegisterRequest` type used by `registerUser` method just to make tests pass

Reply:
What's more, when we use your approach, the names of properties in `registerForm` have to match exactly properties name in `RegisterRequest`. Do we need coupling like this?
I totally agree that your proposition is more elegant but I am not sure if code won't be more fragile

Reply:
Oh well, I didn't expect that parameters will make it through to the actual network request :sweat_smile: Let's keep it as it is then.